pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -784,7 +784,7 @@ private String formatParticipantToStructuredJSON(
 
         // Retrieving information to compute the dsm record
         MedicalRecordService.ParticipantConsents consents = medicalRecordService
-                .fetchBloodAndTissueConsents(handle, participantUser.getId(), userGuid, statusDto.getStudyId(), studyGuid);
+                .fetchBloodAndTissueConsents(handle, participantUser.getId(), userGuid, userGuid, statusDto.getStudyId(), studyGuid);
```

Comment:
Data export is usually ran by background jobs, so there's no operator here.

Reply:
Done